intermediate_html_css/grid/advanced_grid_properties.md

```diff
@@ -306,13 +306,11 @@ And that's about it! Congratulations, if you've made it this far, you are well o
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
-When doing the following exercises, please use all the documentation and resources you need to accomplish them. You are _not_ intended to have any of this stuff memorized at this point. Check the docs, use google, do what you need to do (besides checking the solutions) to get them done.
-
-Go back to our [CSS exercises repository](https://github.com/TheOdinProject/css-exercises) (you've done these previously, but don't forget that the instructions are in the README). Do the exercises in the 'grid' directory in the following order:
-
-1. grid-layout-2
-1. grid-layout-3
-
+1. Review [Part 4 on Grid Properties](https://css-tricks.com/snippets/css/complete-guide-grid/#grid-properties) from CSS-Tricks.
+2. Go back to our [CSS exercises repository](https://github.com/TheOdinProject/css-exercises) (you've done these previously, but don't forget that the instructions are in the README). Do the exercises in the 'grid' directory in the following order:
+ - grid-layout-2
+ - grid-layout-3
+> When doing these exercises, please use all the documentation and resources you need to accomplish them. You are *not* intended to have any of this stuff memorized at this point. Check the docs, use google, do what you need to do (besides checking the solutions) to get them done
```

Comment:
```suggestion
1. Review [Part 4 on Grid Properties](https://css-tricks.com/snippets/css/complete-guide-grid/#grid-properties) from CSS-Tricks.
1. Go back to our [CSS exercises repository](https://github.com/TheOdinProject/css-exercises) (you've done these previously, but don't forget that the instructions are in the README). Do the exercises in the 'grid' directory in the following order:
   - `grid-layout-2`
   - `grid-layout-3`
 
   Note: When doing these exercises, please use all the documentation and resources you need to accomplish them. You are *not* intended to have any of this stuff memorized at this point. Check the docs, use google, do what you need to do (besides checking the solutions) to get them done.
```
To adhere to the style guide. The linter seems to not catch thing because there is no blank line between the opening `<div>` and line 309. If you put a blank line between these two, the linter will kick in. You'll also need to to put a blank line after the note and before the closing `</div>` as part of this.